recipe/meta.yaml

```
@@ -32,8 +32,11 @@ test:
 
 about:
   home: http://www.perl.org/
-  license: Perl Artistic
-  license_family: Other
+  license: Artistic-1.0-Perl
+  license_file:
+    - Artistic  # [unix]
+    - Copying   # [unix]
+    - licenses  # [win]
```

Looks like [the job is failing]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=201721&view=logs&jobId=a70f640f-cc53-5cd3-6cdc-236a1aa90802&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f ). Here's [the raw log attached]( https://github.com/conda-forge/perl-feedstock/files/5138461/perl_win_64_20200827.3_log.txt ). Maybe we can tweak this like so?




```suggestion
    - licenses/perl/*  # [win]
```

---

I sure wouldn't know any better, searching through other conda-forge recipes to see if any others have a complex license_file setup - no luck yet

---

The latest change seems sensible. Was pretty sure directories would be handled correctly, but it looks like I'm wrong.